internal/handlers/pg/msg_serverstatus.go

```diff
@@ -50,7 +50,7 @@ func (h *Handler) MsgServerStatus(ctx context.Context, msg *wire.OpMsg) (*wire.O
 
 	res.Set("catalogStats", must.NotFail(types.NewDocument(
 		"collections", stats.CountCollections,
-		"capped", int32(0), // TODO https://github.com/FerretDB/FerretDB/issues/2342
+		"capped", int32(0),
```

Comment:
The task #2342 is closed, so I guess we don't need to keep this TODO in the old backend.